ruby_on_rails/bootstrap.md

```diff
@@ -30,3 +36,52 @@ config.label_class = 'form-control-label'
 config.input_class = 'form-control'
 ```
 
+To make the error highlighting work you need to add some css to your application
+
+```scss
+.invalid-feedback {
+  display: block;
+}
+
+.has-danger {
+  .form-control {
+    border-color: $form-feedback-invalid-color;
+  }
+}
+```
```

Comment:
Specify that this is a workaround. Related simple form issue: https://github.com/plataformatec/simple_form/pull/1476